engines/scumm/imuse_digi/dimuse_track.cpp

```diff
@@ -356,7 +395,7 @@ Track *IMuseDigital::cloneToFadeOutTrack(Track *track, int fadeDelay) {
 		// so gave up
 		error("Game not supported while playing on 2 different CDs");
 	}
-	track->soundDesc = soundDesc;
+	fadeTrack->soundDesc = soundDesc;
```

Comment:
looks like changing logic here

Reply:
Explaination for the change: 
in a previous row:
`ImuseDigiSndMgr::SoundDesc *soundDesc = _sound->cloneSound(track->soundDesc);`
we are cloning the soundDesc from the old track and storing it in *soundDesc.
I suppose this was done with the intention of assigning it to the fadeTrack (which is the clone currently being constructed).
Was the assignment to track->soundDesc in the old code intentional?

Reply:
yes, I think it was intentional

Reply:
Very well, I'll revert it then.